### DISCUSSION TITLE
langchain如何介入？

### DISCUSSION THREAD

**Discussion by SunLang115** [RapidAI/RapidRAG#7] [Q&A] [answered]
如果使用lanchain做开发，需要在那些地方去修改，能详细讲一下嘛？

**Comment by SWHL** [comment, marked as answer]
我搭建这个框架，本来就是避免引入langchain这个工具的。因为它太不灵活了。
如果想用的话，可以移步到：[langchain-ChatGLM](https://github.com/imClumsyPanda/langchain-ChatGLM)
